app/services/container_dashboard_service.rb

```diff
@@ -100,9 +100,7 @@ def providers
 
   def build_provider_status(ui_type)
     {
-      :iconClass    => "pficon pficon-#{ui_type}",
```

Comment:
where is this code used so this change is ok? (tried to follow it within the patch and didn't find)